data/weapons.txt

```diff
@@ -509,7 +509,7 @@ outfit "Plasma Turret"
 		"shield damage" 63
 		"hull damage" 25
 		"heat damage" 340
-	description "The Plasma Turret is a pair of plasma cannons mounted on a turret so that they can fire in any direction. It is a fearsome weapon, but only the largest of ships have the space and the energy that it requires."
+	description "The Plasma Turret is a pair of plasma cannons mounted on a turret so that they can fire in any direction. It is a fearsome weapon, but only the largest of vessels have the space and the energy that it requires, and its low turn rate makes it best suited to targeting large capital ships."
```

Comment:
shouldn't "to targeting" be "for targeting" (or "to target", but in this sentence it doesn't work)?

Reply:
`to targeting` is fine as used here

Reply:
In terms of prose, I would agree, but I think that actually kind of changes the interpretation of the sentence.
"Best suited to" implies that the best use for Plasma Turrets is targeting capital ships.
"Best suited for" implies that capital ships are best fought using Plasma Turrets.

I would just suggest that we rewrite this section entirely.

Reply:
> "Best suited to" implies that the best use for Plasma Turrets is targeting capital ships.

That's true, though.

I'm open to a total rewrite. Any suggestions?

Reply:
maybe change
"and its low turn rate makes it best suited to targeting large capital ships"
to
"and due to its low turn rate, it's [1 or 2]."
where ya can use either option at the end
option 1: "ineffective against faster targets"
option 2: "only effective against slower ships"

I used "faster" & slower instead of "large capital ships" since it's more about the speed of the enemy ships than its size